include/glow/Graph/Nodes.h

```diff
@@ -130,6 +130,21 @@ class Constant : public Storage {
   bool verify() const;
 };
 
+/// An enum representing the IO policy of the tensor associated with the
+/// placeholder.
+enum class IoPolicy : unsigned char {
+  // Always load, always read
```

Comment:
Period missing at the end of sentence.

Reply:
Also, could you elaborate in the comments what is meant by e.g. load, output and read or read on demand? Load from where (e.g. from host)? Read by whom? I suspect you mean something like:
- InOut is copied from a host to a device and later copied back to the host
- Input does not need to be copied from a host as it is in the device memory already
- Intermediate is probably for activations
- Output is for copying from a device to a host

You may want to make it more clear in the comment.

Also, I suspect that it may have some overlap with @nickgg's work on improving the support for device-specific tensors. Please check with him.